### DISCUSSION TITLE
Is Site URL a comma-separated list?

### DISCUSSION THREAD

**Discussion by Vandivier** [supabase/supabase#3251] [Questions] [answered]
by default, `Site URL` comes with localhost:3000, but I'd like my remote Vercel URL to be able to connect to DB. I've tried two things:
1. Append Site URL value to `http://localhost:3000,https://futurecaster-blitzjs.vercel.app/`
2. Replace localhost with single Site URL being `https://futurecaster-blitzjs.vercel.app/`

my db connection continues to fail with the following error message:

error: Error validating datasource `db`: the URL must start with the protocol `postgresql://` or `postgres://`.\n  -->  schema.prisma:6\n   | \n 5 |   provider = \"postgresql\"\n 6 |   url      = env(\"DATABASE_URL\")\n 

It could be a Vercel-side issue, but starting this discussion just to cover my bases

**Comment by silentworks** [comment, marked as answer]
No the Site URL is not a comma-separated list. The error you are getting wouldn't be related to this though, it seems more like your DB url is malformed.  You can get the DB url from the **settings/database** section of the Supabase dashboard, at the bottom there is a **Connection string** area which has this without the password.
